### DISCUSSION TITLE
Breakout room 11 - Series 3

### DISCUSSION THREAD

**Discussion by morvenjrs** [IBAHCM/RPiR#210] [Archive]
The people in breakout room 11 for series 3 are:

Shaya van Houdt - @shayavh
Sarah Tonnar - @2199105T
Ailsa Watson - @AilsaWatson
Morven Rae-Seaman - @morvenjrs

**Comment by shayavh** [comment]
Link to @shayavh repo is: https://github.com/IBAHCM/shayavhSeries03v

**Comment by 2199105T** [comment]
Link to my repository is: https://github.com/IBAHCM/ST2199105TSeries03

**Comment by morvenjrs** [comment]
My repository can be found at: https://github.com/IBAHCM/morvenjrsSeries03

**Comment by AilsaWatson** [comment]
@morvenjrs Hi Morven could you please add me and my github username to this thread :) Ailsa Watson - @AilsaWatson

**Reply by morvenjrs** [reply]
Updated, sorry I didn't do this sooner!

**Comment by AilsaWatson** [comment]
Hi my Series03 repo link is: 

https://github.com/IBAHCM/AilsaWatsonSeries03.git
